sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedProcessorBuilderImpl.java

```diff
@@ -72,16 +67,13 @@
     private static final String PK_RANGE_ID_SEPARATOR = ":";
     private static final String SEGMENT_SEPARATOR = "#";
     private static final String PROPERTY_NAME_LSN = "_lsn";
-    private static final String PROPERTY_NAME_TS = "_ts";
 
     private final Logger logger = LoggerFactory.getLogger(ChangeFeedProcessorBuilderImpl.class);
-    private static final long DefaultUnhealthinessDuration = Duration.ofMinutes(15).toMillis();
     private final Duration sleepTime = Duration.ofSeconds(15);
     private final Duration lockTime = Duration.ofSeconds(30);
-    private static final int DefaultQueryPartitionsMaxBatchSize = 100;
+    private static final int DEFAULT_QUERY_PARTITIONS_MAX_BATCH_SIZE = 100;
 
-    private int queryPartitionsMaxBatchSize = DefaultQueryPartitionsMaxBatchSize;
-    private int degreeOfParallelism = 25; // default
+    private final static int DEFAULT_DEGREE_OF_PARALLELISM = 25; // default
```

Comment:
The initial intent was to make these settings configurable at some point. However we've been running with these defaults for a while now without any issues and no requests from customers to change them.

Reply:
Yeah - and if necessary this can be wired up if a need arises.